laravel-app/public/assets/js/post_create.js

```diff
@@ -0,0 +1,20 @@
+
+$(document).ready(function() {
+    function previewImage() {
+        var photoInput = document.getElementById('photo');
+        var photoPreview = document.getElementById('photo-preview');
+        var previewImage = document.getElementById('preview-image');
+
+        photoInput.addEventListener('change', function () {
```

Comment:
I suggest next time to utilize the library if you're already using it. If you'll end up using vanilla JS then better to remove it in this use case IMO.

Reply:
Yes sir, I reused `document.addEventListener` and cleaned the code for redundancies